bin/run-dev

```diff
@@ -7,14 +7,21 @@ if [ -f ".secrets" ]; then
     . .secrets
 fi
 
+if [ "$DISABLE_LOCALSTACK" ]; then
```

Comment:
I'd prefer we find another approach here.  I think this is going to add to maintenance burden - we'll need to remember to add any new container to line 12 here, and I think we probably won't.  Is there any other way to do this?

Reply:
Done.

Reply:
Nice, good thought!